censusreporter/apps/census/static/js/comparisons.js

```diff
@@ -990,6 +990,9 @@ function Comparison(options) {
 
             return Bloodhound.tokenizers.whitespace(fields.join(' '));
         },
+        sorter: function(a, b) {
+            return a.table_id.localeCompare(b.table_id);
```

Comment:
It tells the Bloodhound instance how to sort search results. Given two results a and b, return an integer telling it how the two compare. In this case, we want to compare the table ids alphabetically, which are the full names of the tables.

See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort